### PR TITLE
nolo tlc ciel 9_8

### PR DESCRIPTION
**Pull request by nolo16**

### CODE REVIEW

File: script/Ciel/Day 9/09_01_CIEL09_8.txt [hunk 1]
```diff
@@ -273,12 +273,13 @@ Senpai says, with her face facing downward.
...
 -- Page 714, Offset 23992.
 -- なんとなしに言葉がない。
-I don't know why but I find myself lost for words.
+I don't know why, but I'm at a loss for words.
 }
 [sha:1ce0c3c9bd163b8f89146853f60255434d76ae52]{
 -- Page 714, Offset 23993.
 -- 気がつけば体が触れ合うほど近くて、先輩の体温がほのかに感じられて、ムズムズする。
-Without even realizing it, our bodies grow enough to be touching, and Senpai's faint warmth makes me all jittery.
+// sex
```
Comment (Avuxo) [inline script/Ciel/Day 9/09_01_CIEL09_8.txt:281]: is this what sex is? i tried to google it but i couldn't find any info

File: script/Ciel/Day 9/09_01_CIEL09_8.txt [hunk 2]
```diff
@@ -808,7 +810,8 @@ This is―――
 [sha:d85840b030b93fc2ebbea798482998d25ad4ea06]{
 -- Page 754, Offset 24098.
 -- ヴローヴと同じかそれ以上の、蘇生や治癒という表現を超越した、異常な光景だった。
-This is an aberrant sight that goes far beyond the notions of regeneration or healing, one on the level or even above what she displayed against Vlov.
+// not sure if comparing to vlov or battle with vlov, but earlier on in this file its ヴローヴの時 while this is just ヴローヴ
```
Comment (Avuxo) [inline script/Ciel/Day 9/09_01_CIEL09_8.txt:813]: I'm actually not _entirely_ sure, but i guess the last proper subject was 血液が消えていく, so I'd say its the former.

**Review by Avuxo — APPROVED** [review]
kinda only skimmed this one due to time, but didnt see anything super wrong

File: script/Ciel/Day 9/09_01_CIEL09_8.txt (hunk 1 above)
Comment (nolo16) [reply script/Ciel/Day 9/09_01_CIEL09_8.txt:281]: this is what my teachers taught me

**Pull request merged**